stories/alerts/alerts.stories.mdx

```diff
@@ -0,0 +1,136 @@
+import { useState, useEffect, useCallback, useMemo } from "react";
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import { Alerts, pushAlert, clearAlerts } from "@figshare/fcl/alerts";
+import { Button } from "@figshare/fcl/button";
+
+<Meta
+  title="UI/Alerts/Alerts"
+  component={Alerts}
+  parameters={{
+    viewMode: 'docs',
+    previewTabs: {
+      canvas: { hidden: true }
+     },
+  }}
+/>
+
+# Alerts
+---
+- [Overview](#overview)
+- [Props](#props-table)
+- [Examples](#examples)
+
+## Overview
+
+An inline alerts manager component. Holds a list of alert message entries and renders them.
+Messages can be added or removed by dispatching `"form-alerts:message"` type events.
+The module exports utilities to easily dispatch such events.
+
+<Story
+  name="Alerts"
+  parameters={{
+    layout: "centered",
+  }}
+>
+  {() => {
+    return (
+      <div style={{ display: "flex", flexDirection: "column", gap: "12px" }}>
```

Comment:
@ailincaibianca instead of adding duplicate buttons for `Fixed` and `Inline`, just leave a row of buttons for the alerts and add a `Button` to toggle the `isFixed` prop:

```
const [fixed,setFixed] = useState(false);
<div style={{ display: "flex", flexDirection: "row", gap: "12px" }}>
<Button theme="primary" onClick={() => setFixed(!fixed)}>{fixed ? "Show Inline" : "Show fixed to viewport"}</Button>
          <Button theme="secondary" onClick={() => pushAlert({ channel: "fixed-alerts-channel", content: "Something is wrong.", type: "error" })}>Push Error</Button>
          ....
        </div>
        <div className="field-container" style={{ display: "flex", flexDirection: "column", width: "50%" }}>
          <Alerts id="fixed-alerts-channel" isFixed={fixed}/>
        </div>

```